.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+
+    - name: Install bundler
+      run: gem install bundler
+
+    - name: Bundle install
+      run: bundle install
+
+    - name: Run tests
+      run: bundle exec rake parse_data_file
```

Comment:
This doesn't exist yet, right? Should we leave this out until we actually have this rake task and then introduce this check when that's available?

Reply:
@fdiaz I wrote it in a previous PR. https://github.com/airbnb/inclusion/blob/1b4b39eb4ef3993a10693a09796b921ce7875111/Rakefile#L1-L5

I could not run it in CI since you cannot enable GitHub Actions until the repo is public.

Reply:
I will merge but please let me know if I misunderstood your comment @fdiaz and I will fast follow if so.

Reply:
Nope, all good! I don't know why I didn't see the Rakefile at the root of the project 🤷